prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -50,6 +58,17 @@ class FHIRRouter(
             // pull fhir document and parse FHIR document
             val fhirDocument = FhirTranscoder.decode(message.downloadContent())
 
+            // TODO: routing would need to happen here to generate a list of the receivers
+            val hardCodedPhase1ReceiverName = "co-phd.full-elr-hl7"
+            var receiver = settings.findReceiver(hardCodedPhase1ReceiverName)
+            if (receiver == null)
+                logger.error("Receiver $hardCodedPhase1ReceiverName was not found in the settings.")
```

Comment:
You cannot continue the logic at this point because the receiver is null.  Throw exception?

Reply:
Updated